project_euler/problem_145/sol1.py

```diff
@@ -13,44 +13,81 @@
 
 How many reversible numbers are there below one-billion (10^9)?
 """
+EVEN_DIGITS = [digit for digit in range(10) if digit % 2 == 0]
+ODD_DIGITS = [digit for digit in range(10) if digit % 2 == 1]
```

Comment:
```suggestion
EVEN_DIGITS = [0, 2, 4, 6, 8]
ODD_DIGITS = [1, 3, 5, 7, 9]
```

Simpler and readable.

Reply:
Fixed